ts/output/common/Wrappers/scriptbase.ts

```diff
@@ -455,16 +490,25 @@ export function CommonScriptbaseMixin<
       return (node.isToken && (node as MmlMo).getText() === '\u2015');
     }
 
+    /**
+     * @return {number}   The base child's width adjusted for the base italic correction
+     */
+    public getBaseWidth(): number {
+      const bbox = this.baseChild.getBBox();
+      return bbox.w * bbox.rscale - (this.baseRemoveIc ? this.baseIc : 0);
```

Comment:
Is that correct? If we remove the `ic` we take the base value otherwise we take none? Should that be the other way around?

Reply:
No, this is right.  The characters normally include their italic correction (so the bases have italic correction already because that's normally added by TeX except in a few situations).  The `getBaseWidth()` function is supposed to return the width of the base when its italic correction is removed (rather than the actual bounding box size of the base, which includes it).  So that's why it is removed here.

Reply:
I will update the comment above.